.github/workflows/ci-check-repo.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check Formatting and Commiters
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  verify:
+    name: Verify format and commiters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go 1.13
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+        id: go
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check all
+        working-directory: ./go
+        # Keep this in sync with //go/utils/prepr/prepr.sh.
+        run: |
+          go get -mod=readonly ./...
+          ./utils/repofmt/check_fmt.sh
+          ./Godeps/verify.sh
+          ./utils/checkcommitters/check_pr.sh
+          go vet -mod=readonly ./...
+          go run -mod=readonly ./utils/copyrightshdrs/
+          go test -mod=readonly -test.v ./...
```

Comment:
Is it necessary to run the tests here as well as in https://github.com/liquidata-inc/dolt/pull/762 ?

Reply:
No, we don't need to do this here.